astroquery/conftest.py

```diff
@@ -20,25 +19,6 @@
 except NameError:
     pass
 
-
-# pytest magic:
-# http://pytest.org/latest/plugins.html#_pytest.hookspec.pytest_configure
-# use pytest.set_trace() to interactively inspect config's features
-def pytest_configure(config):
-    if config.getoption('remote_data'):
-        pass
-    else:
-        turn_off_internet(verbose=config.option.verbose)
```

Comment:
@keflavich - Having a second look, maybe we can also remove turn_off/on_internet from ``astroquery/utils/testing_tools.py`` as it also got merged upstream into ``astropy/tests/disable_internet.py`` and isn't used anywhere in the code anyway.